app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -0,0 +1,55 @@
+module MiddlewareServerHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    @prop_hash = eval(@record.properties)
+
+    %i(name host bind_addr product version)
+  end
+
+  def textual_group_relationships
+    # Order of items should be from parent to child
+    %i(ems middleware_deployments)
+    # items.concat(%i(middleware_servers))
+    # items
+  end
+
+  def textual_group_smart_management
+    %i(tags)
+  end
+
+  #
+  # Items
+  #
+
+  def textual_name
+    @record.name
+  end
+
+  def textual_host
+    @record.host
+  end
+
+  def textual_bind_addr
```

Comment:
I am not able to get this working and I wonder how others do it - especially as I see 

```
 def textual_ipaddress
    return nil if @ems.kind_of?(ManageIQ::Providers::Amazon::CloudManager)
    {:label => "Discovered IP Address", :value => @ems.ipaddress}
  end
```
In `EmsCouldHelper::TextualSummary` which also seems to not use the `ipaddress` defined in `en.yml`